app/views/contexts/summary/list_collector_block.py

```diff
@@ -92,7 +96,9 @@ def list_summary_element(self, summary: Mapping[str, Any]) -> dict[str, Any]:
             remove_block_id = list_collector_block["remove_block"]["id"]
             add_link = self._add_link(summary, list_collector_block)
             repeating_blocks = list_collector_block.get("repeating_blocks", [])
-            related_answers = self._get_related_answers(current_list, repeating_blocks)
+            related_answers = self._get_related_answers(
+                list_model=current_list, repeating_blocks=repeating_blocks
```

Comment:
Non blocker, If list_model is an instance of `ListModel` class maybe we should rename the var in get_related_answers so it is `current_list` too or rename `current_list` to `list_model`, just for clarity?
Especially when we call list store below and var is list_model again:
```python
     list_model = self._list_store[list_name]
```

Reply:
Yeah that seems like a good switch 👍 changed to use `list_model` as that matches the other stuff there better I think